src/v2/components/BlockLightbox/components/BlockLightboxText/index.js

```diff
@@ -20,7 +20,15 @@ export default class BlockLightboxText extends PureComponent {
     mode: 'resting',
   }
 
-  openModal = () => this.setState({ mode: 'editing' })
+  openModal = () => {
+    if (
+      window.getSelection().toString() ||
+      window.getSelection().toString() !== ''
+    ) {
+      return false
+    }
+    this.setState({ mode: 'editing' })
+  }
```

Comment:
while ur in here wanna cover link clicks too?

Reply:
sure